modules/ipi-install-network-requirements.adoc

```diff
@@ -15,18 +15,15 @@ Installer-provisioned installation of {product-title} involves several network r
 +
 In {product-title} 4.3, when deploying using the `provisioning` network, the first NIC on each node, such as `eth0` or `eno1`, must interface with the `provisioning` network.
 +
-In {product-title} 4.4 and later releases, you can specify the provisioning network NIC with the `provisioningNetworkInterface` configuration setting.
+In {product-title} versions 4.4 through 4.8, you can specify the `provisioning` network NIC with the `provisioningNetworkInterface` configuration setting.
```

Comment:
I see this file has already been doing it like this, but this looks a little odd to be talking about how something is done in 4.3, and 4.4-4.8, when you're in the 4.9 docs. That older info should really just be in the older versions, and only show what is applicable for 4.9 in the 4.9 docs. Maybe something you can consider cleaning up in a future release?

Reply:
To follow up on this again, anything before 4.6 is unsupported anyways, so it really doesn't seem to make sense to have that in

Reply:
I omitted 4.3, and references to 4.4. Since this is a significant change to the networking, I think understanding the difference between 4.8 and 4.9 behavior would be useful for those upgrading.